src/EditorFeatures/CSharpTest/EditAndContinue/StatementEditingTests.cs

```diff
@@ -2761,6 +2761,78 @@ void F()
                 Diagnostic(RudeEditKind.ChangingLambdaReturnType, "a", CSharpFeaturesResources.lambda));
         }
 
+        [Fact]
+        public void Lambdas_Update_Signature_NullableResult()
+        {
+            var src1 = @"
+using System;
+
+class C
+{
+    void G1(Func<string, string> f) {}
+    void G2(Func<string, string?> f) {}
+
+    void F()
+    {
+        G1(a => a);
+    }
+}
+";
+            var src2 = @"
+using System;
+
+class C
+{
+    void G1(Func<string, string> f) {}
+    void G2(Func<string, string?> f) {}
+
+    void F()
+    {
+        G2(a => a);
+    }
+}
+";
+            var edits = GetTopEdits(src1, src2);
+
+            edits.VerifySemanticDiagnostics();
+        }
+
+        [Fact]
+        public void Lambdas_Update_Signature_NullableInput()
```

Comment:
>Lambdas_Update_Signature_NullableInput [](start = 20, length = 38)

I think a single test that changes both types would be sufficient.

Reply:
Fixed. Thanks!